.infrastructure/deployment.yml

```diff
@@ -52,3 +66,13 @@ spec:
             port: 8080
           initialDelaySeconds: 5
           periodSeconds: 5
+      volumes:
+        - name: data
+          persistentVolumeClaim:
+            claimName: pvc-data
+        - name: app-secrets-volume
+          secret:
+            secretName: app-secret
+        - name: app-config-volume
+          secret:
+            secretName: app-config
```

Comment:
The `app-config-volume` is incorrectly using `secret` for mounting. It should be using `configMap` instead, as the `app-config` is a ConfigMap, not a secret. Please update the volume definition to use `configMap`.